tests/ttnn/unit_tests/operations/test_matmul.py

```diff
@@ -497,3 +497,80 @@ def test_falcon_query_key_value_matmul(device, batch_size, m_size, k_size, n_siz
 
     output_tensor = ttnn.to_torch(output_tensor)
     assert_with_pcc(torch_output_tensor, output_tensor, pcc=0.996)
+
+
+@pytest.mark.parametrize(
+    "batch_size, channel_a, channel_b, m_size, k_size, n_size, has_bias",
+    [
+        (1, 2, 1, 1024, 640, 2560, False),
+        (2, 8, 8, 64, 96, 160, False),
+        (1, 2, 1, 4096, 320, 1280, False),
+        (1, 2, 1, 64, 1280, 5120, False),
+        (2, 8, 8, 64, 64, 160, False),
+        (1, 2, 1, 1024, 640, 768, False),
+        (2, 8, 8, 96, 160, 96, False),
+        (2, 8, 8, 1024, 1024, 96, False),
+        (1, 2, 1, 96, 768, 1024, False),
+        (1, 1, 1, 32, 1280, 1280, True),
+        (2, 8, 8, 4096, 96, 64, False),
+        (1, 2, 1, 64, 5120, 1280, True),
+        (2, 8, 8, 4096, 64, 96, False),
+        (1, 2, 1, 1024, 768, 640, True),
+        (1, 2, 1, 256, 1280, 1280, True),
+        (2, 8, 8, 1024, 96, 96, False),
+        (1, 2, 1, 1024, 640, 2304, False),
+        (1, 1, 1, 32, 1280, 320, True),
+        (1, 2, 1, 96, 768, 2560, False),
+        (1, 2, 1, 4096, 1280, 320, True),
+        (1, 2, 1, 1024, 2560, 640, True),
+        (1, 2, 1, 256, 1280, 3840, False),
+        (1, 1, 1, 32, 320, 1280, True),
+        (1, 2, 1, 4096, 512, 320, True),
+        (1, 2, 1, 64, 1280, 1280, True),
+        (1, 2, 1, 256, 5120, 1280, True),
+        (1, 2, 1, 256, 1280, 1280, False),
+        (2, 8, 8, 256, 160, 96, False),
+        (2, 8, 8, 256, 256, 160, False),
+        (1, 2, 1, 96, 768, 1536, False),
+        (1, 2, 1, 64, 1280, 3840, False),
+        (2, 8, 8, 1024, 96, 1024, False),
+        (2, 8, 8, 256, 96, 160, False),
+        (1, 2, 1, 64, 1280, 1280, False),
+        (2, 8, 8, 4096, 64, 4096, False),
+        (1, 1, 1, 32, 1280, 640, True),
+        (2, 8, 8, 64, 160, 64, False),
+        (1, 2, 1, 4096, 320, 1536, False),
+        (1, 2, 1, 256, 1280, 5120, False),
+        (2, 8, 8, 4096, 4096, 64, False),
+        (2, 8, 8, 256, 160, 256, False),
+        (1, 2, 1, 4096, 320, 512, False),
+    ],
+)
+# @pytest.mark.parametrize("core_grid", [None, ttnn.CoreGrid(8, 8)])
+def test_sd_matmul(device, batch_size, channel_a, channel_b, m_size, k_size, n_size, has_bias):  # , core_grid):
```

Comment:
Can you please also create tests for bfp8 as well?

Reply:
sure, will do. Do we actually need bfloat16 tests or just having cfp8-b would be enough?

Reply:
let's have both. You can just add a pytest parameter for it.